src/main/java/net/datafaker/providers/base/Aws.java

```diff
@@ -59,21 +59,25 @@ public String route53ZoneId() {
     }
 
     public String securityGroupId() {
-        return "sg-" + randHex();
+        return "sg-" + randHex(17);
     }
 
     public String subnetId() {
-        return "subnet-" + randHex();
+        return "subnet-" + randHex(17);
     }
 
     public String vpcId() {
-        return "vpc-" + randHex();
+        return "vpc-" + randHex(17);
     }
 
     private String appName() {
         return faker.app().name().toLowerCase().replaceAll("\\W+", "");
     }
 
+    private String randHex(int length) {
+        return faker.random().hex(length, false);
+    }
+
     private String randHex() {
         return faker.random().hex(16, false);
```

Comment:
I would suggest to replace then the implementation of `randHex` with something like
```java
private String randHex() {
        return randHex(16);
}        
```